src/main/kotlin/no/nav/familie/ef/sak/opplysninger/personopplysninger/dto/PersonopplysningerDto.kt

```diff
@@ -38,6 +39,8 @@ data class BarnDto(
     val annenForelder: AnnenForelderMinimumDto?,
     val adresse: List<AdresseDto>,
     val borHosSøker: Boolean,
+    val deltBosted: List<DeltBosted>,
```

Comment:
Burde vi ha et eget objekt for DeltBosted og ikke bruke PDL-objektet? Sånn at vi får tilpasset det likt de andre delene som sendes ut til frontend for `Personopplysninger` ?